### ISSUE TITLE
Integration with Redmine [feature request]

### ISSUE THREAD

**Issue by buscon** [TailorDev/Watson#348]
hi,

it would be great to be able to log the time tracked by watson into [Redmine](https://redmine.org/).

thanks!

**Comment by jmaupetit** [comment]
Hi @buscon! I'm not familiar with redmine, do you have resources to share about such integration?

**Comment by buscon** [comment]
yes, sure. in Redmine you can log time for a specific issue using their REST API, here are more information:
https://www.redmine.org/projects/redmine/wiki/Rest_TimeEntries#Creating-a-time-entry

As you already plan to sync the track time with a server, I imagine it would not be difficult. you just have to send the tracked time to a specific issue (#NUMBER) through the REST API.

thank you!
